### DISCUSSION TITLE
Project status and pending PR inquiry for ublox integration

### DISCUSSION THREAD

**Discussion by gshubham96** [ublox-rs/ublox#82] [General]
We're considering integrating the ublox crate with our project stack and noticed that there are some pending pull requests in the repository. We wanted to ask the community if this project is still being actively maintained, or if it has been abandoned. Any updates on the status of the pending pull requests would be greatly appreciated as they could help us make an informed decision on whether to move forward with the integration.

Looking forward to any information you can provide.

Thanks in advance!

**Comment by gwbres** [comment]
the project is far from abandonned, it is sill maintained and will remain as is.
we're just all busy with our daily jobs and merge when we get time to do so. 
I am sure we will accept new members as well to help the effort

**Reply by gshubham96** [reply]
Hey
Thanks for your response! It was helpful. :)

Regards

**Reply by gwbres** [reply]
BTW, anyone can contribute and contributions can have several forms..
Like solving pending issues.. As far as i'm concerned, I need to review the Z9P request and I don't have much time to dedicate to open source projects for about 1 or 2 months. The little time I have is consumed by my gnss libraries, but once this phase is over, I will get back to this crate, for the simple reason the gnss tools will finally use it
